client/package.json

```diff
@@ -16,8 +16,6 @@
     "compile": "lerna run --scope @openmsupply-client/* --parallel tsc --since HEAD -- --noEmit",
     "re-compile": "tsc --project packages/host/tsconfig.json --noEmit",
     "compile-full": "lerna run --scope @openmsupply-client/* --parallel tsc",
```

Comment:
Just wondering what we want "re-compile" for, as opposed to "compile-full"?

Reply:
We don't want any parallel compilation it's 3 x slower